.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: build
+on:
+  push:
+    branches:
+      - '*'
+  pull_request: {}
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - run: "go vet ./..."
+      - name: Lint
+        uses: dominikh/staticcheck-action@v1.2.0
+        with:
+          version: "2022.1.1"
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - run: "make test"
+  snapshot-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - name: Set GOLANG_VERSION
+        run: |
+          echo "GOLANG_VERSION=$(go version | awk '{ print $3 }')" >> $GITHUB_ENV
+      - name: Snapshot Build
+        uses: goreleaser/goreleaser-action@v3
```

Comment:
We are also on `goreleaser-action@v4` now, remove `--debug` flag (https://goreleaser.com/deprecations/#-debug)